crates/op-rbuilder/src/builders/context.rs

```diff
@@ -517,21 +517,37 @@ impl OpPayloadBuilderCtx {
             info.executed_transactions.push(tx.into_inner());
         }
 
+        let payload_tx_simulation_time = execute_txs_start_time.elapsed();
         self.metrics
             .payload_tx_simulation_duration
-            .record(execute_txs_start_time.elapsed());
+            .record(payload_tx_simulation_time);
+        self.metrics
+            .payload_tx_simulation_gauge
+            .set(payload_tx_simulation_time);
         self.metrics
             .payload_num_tx_considered
             .record(num_txs_considered as f64);
+        self.metrics
+            .payload_num_tx_considered_gauge
+            .set(num_txs_considered as f64);
         self.metrics
             .payload_num_tx_simulated
             .record(num_txs_simulated as f64);
+        self.metrics
+            .payload_num_tx_simulated_gauge
+            .set(num_txs_simulated as f64);
         self.metrics
             .payload_num_tx_simulated_success
             .record(num_txs_simulated_success as f64);
+        self.metrics
+            .payload_num_tx_simulated_success_gauge
+            .set(num_txs_simulated_success as f64);
         self.metrics
             .payload_num_tx_simulated_fail
             .record(num_txs_simulated_fail as f64);
+        self.metrics
+            .payload_num_tx_simulated_fail_gauge
+            .set(num_txs_simulated_fail as f64);
         self.metrics
             .bundles_reverted
             .record(num_bundles_reverted as f64);
```

Comment:
there are a lot of metrics getting recorded here, consider moving to a separate function